app/controllers/admin/export/document_controller.rb

```diff
@@ -70,4 +70,18 @@ def page_number
   def items_per_page
     params.fetch(:page_count, 100)
   end
+
+  def document_type
+    @document_type ||= Whitehall.edition_classes.find do |type|
+      type.name.underscore == params.require(:type)
+    end
+  end
+
+  def document_subtypes
+    return unless document_type.respond_to?(:subtypes) && params[:subtypes]
+
+    document_type.subtypes.select do |subtype|
+      params[:subtypes].include?(subtype.key)
```

Comment:
Might want to do `Array(params[:subtypes])` to deal with subtypes passed in not as an array (as we don't use strong parameters to enforce that).

Reply:
Good point, I've made this change